src/posts/xarray-biology/index.md

```diff
@@ -0,0 +1,135 @@
+
+That said, many parts of biology generate array data. And scientists in some areas are already benefiting from Xarray.
+
+### Who is Using Xarray
+
+- [`Sgkit`](https://sgkit-dev.github.io/sgkit/latest/getting_started.html#data-structures) “a toolkit for quantitative and population genetics” uses `Xarray` `Dataset`s as a core data structure. This allows them to keep track of the rich metadata associated with sequences and further uses computational patterns to speed up analyses.
+- `Xarray` has also been used for neurophysiology data by the Ecephys package. See a previous blog post [xarray for neurophysiology](https://xarray.dev/blog/xarray-for-neurophysiology) to learn more.
+- The `scverse` family of software uses `anndata`, which, like `Xarray` provides labeled dimensions.
+- The `bioio` package, capable of reading a zoo of microscope formats, provides a function to get image stacks data as Xarray object.
+- The `Xarray.DataTree` provides a natural representation of multiscale images and can interoperate well with the OME-NGFF model. The [multiscale-spatial-image](https://github.com/spatial-image/multiscale-spatial-image?tab=readme-ov-file#multiscale-spatial-image) project use DataTree for precisely this purpose.
+
+## What has limited adoption by Biologists?
+
+Given the benefits of switching to `Xarray`, why aren’t more biologists using it? Is it secretly not as good as this blog claims?
+
+As a long-time user of `Xarray` for microscopy data, I had the same question, so I spent February and April 2025 reaching out to scientists and scientific software developers across various fields of biology to answer it. From those discussions, I’ve found that three core reasons have prevented biologists who are already using Python from adopting `Xarray`.
+
+### Awareness and Examples
+
+Many biologists have just never heard of `Xarray`. This is because the geosciences community created Xarray, and there wasn’t enough community overlap for the word to spread to biologists.
+
+A related issue is that the examples and tutorials are geoscience-centric. I admit that when I first read the xarray docs, I was discouraged and thought it wouldn’t work for my microscopy data. Fortunately, I eventually rediscovered Xarray, mentally translated the examples to biology, and found it incredibly useful.
+
+### Technical Barriers
+
+Once a potential user is convinced of `Xarray`'s value, they may still face technical barriers. Ranging from rough edges to missing features, however, none are insurmountable. An example of a rough edge is that, as of May 2025, you cannot use integers as keys in a `DataTree`. That is a problem, as integers are a natural key to use when tracking single cell lineages. Rough edges like this one haven’t been smoothed over yet because there has not been a user base of biologists using `Xarray`, discovering them, and raising issues to get them fixed.
```

Comment:
This should be two paragraphs. One introducing the idea that technical limitations might be preventing use by bio people, then start enumerating examples in a separate paragraph / list.